acstools/acscte.py

```diff
@@ -1,6 +1,7 @@
 """
-The acscte module contains a function `acscte` that calls the ACSCTE executable.
-Use this function to facilitate batch runs of ACSCTE, or for the TEAL interface.
+The acscte module contains the functions `acscte` and `acscte_forward_model` that call the ACSCTE
+and ACSCTEFORWARDMODEL executables respectively.
+Use the `acscte` function to facilitate batch runs or for the TEAL interface.
```

Comment:
Please clarify whether the batch run is for both or only ACSCTE.

Please add an example of its usage in Python a few lines below.

Reply:
>Please clarify whether the batch run is for both or only ACSCTE.

I thought "Use _this_ function" -> "Use the `acscte` function" was clarification, no?

>Please add an example of its usage in Python a few lines below.

 Anyone...? 😉